modules/examples/eb-sf-batch/modulestack.yaml

```diff
@@ -25,7 +28,11 @@ Resources:
               - "s3:Get*"
               - "s3:List*"
             Effect: Allow
-            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+            Resource: 
+              - !Sub 
+                - "arn:aws:s3:::${ProjectName}*"
+                - ProjectName: !Ref ProjectName
```

Comment:
Is this correct? can it be one-liner ie
```
- !Sub "arn:aws:s3:::${ProjectName}*"
``

Reply:
Yea they both work, changed it for readability.